js/components/report/iframe-answer.js

```diff
@@ -36,22 +32,49 @@ export default class IframeAnswer extends PureComponent {
     */
     try {
       const json = JSON.parse(answer);
-      const interactiveState = JSON.parse(json.interactiveState);
-      return interactiveState.lara_options.reporting_url;
+      const interactiveState = json.interactiveState ? JSON.parse(json.interactiveState) : null;
+      if (interactiveState && interactiveState.lara_options && interactiveState.lara_options.reporting_url) {
+        return interactiveState.lara_options.reporting_url;
+      }
+      return null;
     } catch (e) {
       return answer;
     }
   }
 
+  /**
+   * Adds a studentId and iframeQuestionId to the existing url
+   */
+  getStandaloneLinkUrl(question, answer) {
+    const baseUrl = `${window.location.origin}${window.location.pathname}`;
+    const params = queryString.parse(window.location.search);
+    params.studentId = answer.get("platformUserId");
+    params.iframeQuestionId = question.get("id");
+    const newSearch = queryString.stringify(params);
+    return `${baseUrl}?${newSearch}`;
+  }
+
   renderLink() {
     const { answer, question } = this.props;
+    const { iframeVisible } = this.state;
     const linkUrl = this.getLinkURL(answer.get("answer"));
-    const decorator = question.get("displayInIframe") ? "" : <span className="pr-icon-external-link" />;
-    return <a href={linkUrl} onClick={this.toggleIframe} target="_blank">View Work {decorator}</a>;
+    const externalLinkIcon = <span className="pr-icon-external-link" />;
+    if (question.get("displayInIframe")) {
+      const toggleText = iframeVisible ? "Hide" : "View Work";
+      const standaloneLinkUrl = this.getStandaloneLinkUrl(question, answer);
+      return (
+        <React.Fragment>
+          <a onClick={this.toggleIframe} target="_blank">{toggleText}</a> |{" "}
+          <a href={standaloneLinkUrl} target="_blank">Open in new tab {externalLinkIcon}</a>
+        </React.Fragment>
+      );
+    } else {
+      return <a href={linkUrl} target="_blank">View work in new tab {externalLinkIcon}</a>;
+    }
   }
 
   renderIframe() {
-    const { answer, question, alwaysOpen } = this.props;
+    const { answer, question } = this.props;
```

Comment:
Apparently this is untested too. I guess the cypress tests are not toggling iframes which I think would explain why this is untested. Adding that test would be great.